Resizin/Classes/ResizinSettings.swift

```diff
@@ -59,6 +59,14 @@ public struct ResizinSettings {
         case right = "90"
         case down = "180"
     }
+    
+    /// Returns the image transformed into the selected format
+    public enum OutputFormat: String {
+        case jpeg
+        case png
+        case tiff
+        case webp
```

Comment:
❓ since `webp` is on iOS supported from iOS 14, maybe we should make it usable only on iOS 14? I think that `@available` directive should work on enum cases.

Well it might not make sense to limit `OutputFormat` usage this way as Resizin just constructs correct URL, but still it might be good point for discussion.

Reply:
Good point 👍